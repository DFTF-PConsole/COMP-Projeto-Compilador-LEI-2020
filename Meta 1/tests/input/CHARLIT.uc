'a'	asdasd
'\n'
'\t'
'\\'
'\''
'\"'
'\1'
'\a'
'ç'
'a
'é
'éça
'\12'
'\755'
'\999'	AAA
'\\
'\'
'aaaaaa'
'\\\\' asdadsadsa
'\\'
'\'
'a' asdasdad
'\\'
