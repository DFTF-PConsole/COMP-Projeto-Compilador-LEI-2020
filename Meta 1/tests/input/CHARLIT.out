CHRLIT('a')
ID(asdasd)
CHRLIT('\n')
CHRLIT('\t')
CHRLIT('\\')
CHRLIT('\'')
CHRLIT('\"')
CHRLIT('\1')
Line 8, col 1: invalid char constant ('\a')
Line 9, col 1: invalid char constant ('รง')
Line 10, col 1: unterminated char constant
Line 11, col 1: unterminated char constant
Line 12, col 1: unterminated char constant
CHRLIT('\12')
CHRLIT('\755')
Line 15, col 1: invalid char constant ('\999')
ID(AAA)
Line 16, col 1: unterminated char constant
Line 17, col 1: unterminated char constant
Line 18, col 1: invalid char constant ('aaaaaa')
Line 19, col 1: invalid char constant ('\\\\')
ID(asdadsadsa)
CHRLIT('\\')
Line 21, col 1: unterminated char constant
CHRLIT('a')
ID(asdasdad)
CHRLIT('\\')
